src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2362,7 +2362,7 @@ def save_lora_weights(
 
 class CogVideoXLoraLoaderMixin(LoraBaseMixin):
     r"""
-    Load LoRA layers into [`CogVideoXTransformer3DModel`]. Specific to [`CogVideoX`].
+    Load LoRA layers into [`CogVideoXTransformer3DModel`]. Specific to [`CogVideoXPipeline`].
```

Comment:
Unrelated change but doesn't hurt I guess.

Reply:
Looks okay to fix here